app/models/token.py

```diff
@@ -0,0 +1,21 @@
+import uuid
+from datetime import datetime
+
+from sqlmodel import SQLModel
+
+
+class TokenModel(SQLModel):
+    token: str
+
+    @classmethod
+    def from_str(cls, key: str) -> "TokenModel":
+        return cls(token=key)
+
+
+class TokenPayload(SQLModel):
+    sub: str
+    exp: datetime
+    iat: datetime
```

Comment:
Que significan las variables sub y iat? son acrónimos de que palabras?
Si no es muy larga, tal vez conviene dejarla con un nombre más explícito.
```suggestion
    sub: str
    expire_date_time: datetime
    iat: datetime
```

Reply:
No se pueden renombrar ya que esos nombres son el standar de [RFC 7519](https://datatracker.ietf.org/doc/html/rfc7519) 
los valores son:

- sub (subject): Representa al usuario o entidad para quien se emite el token.
- exp (expiration): Indica cuándo expira el token.
- iat (issued at): Cuándo se generó el token.

La biblioteca de JWT usa esos nombres por defecto para renombrarlos habria que hacer unas cuantas conversiones intermedias y al final habria que convertirlos a esos nombres